minecraft_servers/pvpacademy/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "server_name": "pvpacademy",
+  "nice_name": "PvPAcademy",
+  "direct_ip": "pvpacademy.net",
+  "server_wildcards": [
+    "%.pvpacademy.net"
+  ],
+  "social": {
+    "web": "https://pvpacademy.net",
+    "web_support": "https://pvpacademy.net/discord",

```

Comment:
"web_support" cannot be a link to a discord. You need a faq page or similar things here. See here: https://github.com/LabyMod/server-media/pull/851#discussion_r1428335162